api/v1/serializers/pets-serializer.js

```diff
@@ -5,7 +5,7 @@ const _ = require('lodash');
 
 const { serializerOptions } = appRoot.require('utils/jsonapi');
 const { openapi } = appRoot.require('utils/load-openapi');
-const { paginate } = appRoot.require('utils/paginator');
+//const { paginate } = appRoot.require('utils/paginator');
```

Comment:
Looks like this change was included by accident. This is causing a linter error:
```
   8:1   error  Expected exception block, space or tab after '//' in comment  spaced-comment
  43:22  error  'paginate' is not defined                                     no-undef
```